### ISSUE TITLE
EKS - log: Fluent-Bit Deployment Configuration Generation Failure on EKS Versions Beyond v1.24

### ISSUE THREAD

**Issue by YikaiHu** [aws-solutions/centralized-logging-with-opensearch#133]
### Describe the bug

Fluent-Bit Deployment Configuration Generation Failure on EKS Versions Beyond v1.24, both in DeamonSet and SideCar model.

### Expected Behavior

Generate the Fluent-Bit deployment configuration successfully.

### Current Behavior

An error is thrown by the frontend when clicking on the DeamonSet page.

### Reproduction Steps

Import an EKS with version higher than v1.24.

Create an app log ingestion.

Visit the fluent-bit configuration guide page.

### Solution Version

v1.0.1

**Comment by YikaiHu** [comment]
Working on this, we will create a patch on this.

**Comment by stssp86** [comment]
Thank you, I'm using EKS 1.27 and have this exact problem.

**Comment by YikaiHu** [comment]
Hi @stssp86 , here's a method for quickly fixing the issue manually.

1. Go to AWS Lambda Console
2. Search for `EKS Cluster DaemonSet And Sidecar Config APIs Resolver` Function

![image](https://github.com/aws-solutions/centralized-logging-with-opensearch/assets/34271744/80912efa-7795-408d-a03d-fedcfa996126)

3. Go to `util` -> `log_agent_helper.py`
Replace the `version_map` with the value below, it is at line 1311 :

```python
            version_map = {
                "1.19": "1.19.6/2021-01-05",
                "1.20": "1.20.15/2022-10-31",
                "1.21": "1.21.14/2023-01-30",
                "1.22": "1.22.17/2023-05-11",
                "1.23": "1.23.17/2023-05-11",
                "1.24": "1.24.13/2023-05-11",
                "1.25": "1.25.9/2023-05-11",
                "1.26": "1.26.4/2023-05-11",
                "1.27": "1.27.1/2023-04-19",
            }
```
4. Deploy the Lambda
5. Refresh the Centralized Logging with OpenSearch Solution console.

![image](https://github.com/aws-solutions/centralized-logging-with-opensearch/assets/34271744/7d804470-2ce0-405a-b1ef-5dd3624cb479)

**Comment by stssp86** [comment]
Hello @YikaiHu ,Thank you very much your method for quick fix is work and the DeamonSet page can generate deployment. I'm able to deploy fluent-bit.
![Screenshot 2023-06-20 171852](https://github.com/aws-solutions/centralized-logging-with-opensearch/assets/122448333/bb925e01-72d3-491e-a2cb-ebd92f6b74fe)

**Comment by JoeShi** [comment]
The issue has been fixed in PR #134, please update your solution version to v1.0.2 using the new template 
```
https://s3.amazonaws.com/solutions-reference/centralized-logging-with-opensearch/v1.0.2/<template-name>.template
```
